services/rook-ceph/1.10.1/rook-ceph-helmrelease.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
```

Comment:
Can we move this into a dir and put it behind another flux kustomization just in case we need to define jobs on top of this in the future? (e.g. upgrade strategies)